src/mcts/stoppers/alphazero.cc

```diff
@@ -36,33 +36,49 @@ class AlphazeroTimeManager : public TimeManager {
   AlphazeroTimeManager(int64_t move_overhead, const OptionsDict& params)
       : move_overhead_(move_overhead),
         alphazerotimepct_(
-            params.GetOrDefault<float>("alphazero-time-pct", 12.0f)) {
+            params.GetOrDefault<float>("alphazero-time-pct", 12.0f)),
+        alphazeroincrementpct_(
+            params.GetOrDefault<float>("alphazero-increment-pct", 95.0f)) {
     if (alphazerotimepct_ < 0.0f || alphazerotimepct_ > 100.0f)
       throw Exception("alphazero-time-pct value to be in range [0.0, 100.0]");
+    if (alphazeroincrementpct_ < 0.0f || alphazeroincrementpct_ > 100.0f)
+      throw Exception(
+          "alphazero-increment-pct value to be in range [0.0, 100.0]");
   }
   std::unique_ptr<SearchStopper> GetStopper(const GoParams& params,
                                             const NodeTree& tree) override;
 
  private:
   const int64_t move_overhead_;
   const float alphazerotimepct_;
+  const float alphazeroincrementpct_;
 };
 
 std::unique_ptr<SearchStopper> AlphazeroTimeManager::GetStopper(
     const GoParams& params, const NodeTree& tree) {
   const Position& position = tree.HeadPosition();
   const bool is_black = position.IsBlackToMove();
   const std::optional<int64_t>& time = (is_black ? params.btime : params.wtime);
+  const std::optional<int64_t>& inc = (is_black ? params.binc : params.winc);
```

Comment:
No need it to be a reference, temporary object will be created anyway for std::optional (also in the line above)